lib/jsonapi/resource_common.rb

```diff
@@ -1229,7 +1237,7 @@ def _clear_attribute_options_cache
       end
 
       def _clear_model_to_resource_type_cache
```

Comment:
Rename this to align with the rest of the name changes?

Reply:
I thought about it, but the reason I didn't is because this already didn't align, it just exposes that there is a cache, but doesn't leak the difference between `model_class` and `model_class_name` even before my change, and there's no reason to do so. I updated the internal variable for clarity, but external to `ResourceCommon` you just need to know that there's a cache and you can clear it.

Happy to reconsider if you see it differently.